public/app/plugins/panel/timeline/timeline.ts

```diff
@@ -33,12 +31,13 @@ export interface TimelineCoreOptions {
   numSeries: number;
   rowHeight: number;
   colWidth?: number;
+  theme: GrafanaTheme2;
   showValue: BarValueVisibility;
   isDiscrete: (seriesIdx: number) => boolean;
-
+  colorLookup: (seriesIdx: number, value: any) => string;
```

Comment:
added this because I needed a plain string color for to know what the value text should be